configure.ac

```diff
@@ -3,13 +3,14 @@
 
 AC_PREREQ([2.69])
 AC_INIT([sai-vm],[1.0])
-AM_INIT_AUTOMAKE([subdir-objects])
+AM_INIT_AUTOMAKE([subdir-objects foreign])
 AC_CONFIG_SRCDIR([src])
 AC_CHECK_HEADERS([libxml/parser.h])
 
 # Checks for programs.
 AC_PROG_CXX
 AC_PROG_CC
+AC_DISABLE_STATIC
 LT_INIT([shared])
```

Comment:
LT_INIT([shared]) should already do the same thing as AC_DISABLE_STATIC.  It's not clear to me why it's needed.